src/botPage/view/blockly/index.js

```diff
@@ -317,6 +317,31 @@ export default class _Blockly {
     load(blockStr = '', dropEvent = {}) {
         let xml;
 
+        const validateXML = xml => {
+            if (document.implementation.createDocument) {
+                let xmlDoc;
+
+                try {
+                    xmlDoc = new DOMParser().parseFromString(xml, 'application/xml');
```

Comment:
We can define `xmlDoc` as `const` here and move the logic for checking whether there was a `parsererror` inside the `try` statement. (Starting from line 330)